content/doc/book/system-administration/monitoring.adoc

```diff
@@ -15,3 +12,27 @@ ifdef::backend-html5[]
 endif::[]
 
 = Monitoring Jenkins
+
+NOTE: This page is under development, there will be more content added soon.
+See the jira:WEBSITE-738[] EPIC for tasks related to this page, contributions are welcome!
+
+== Monitoring with Datadog
+
+- plugin:datadog[Datadog plugin for Jenkins]
+- plugin:metrics-datadog[Metrics-Datadog plugin for Jenkins]
+- https://www.datadoghq.com/blog/monitor-jenkins-datadog[Jenkins on Datadog]
+
+== Monitoring with Prometheus and Grafana
+
+- plugin:prometheus[Prometheus plugin for Jenkins]
+- https://medium.com/@eng.mohamed.m.saeed/monitoring-jenkins-with-grafana-and-prometheus-a7e037cbb376[How-To blog on Medium]
+
+== Monitoring with JavaMelody
+
+- plugin:monitoring[Monitoring plugin for Jenkins]
+
+== Other Monitoring Plugins
+- plugin:versioncolumn[Versions Node Monitors]
+- plugin:systemloadaverage-monitor[Agents Monitoring for unix nodes]
+- plugin:network-monitor[Monitor for network connectivity]
```

Comment:
I'd remove this one.  It last released 9 years ago and only monitors DNS name resolution of two fixed names.  Has no user interface to configure it.  Checks that the agent can resolve `www.sun.com` and `www.google.com`.
```suggestion
```